File: 20200406/code.py
def solution():
  expression = input().split("-")
  sum = 0

  for i in expression[0].split("+"):
    sum += int(i)
    
  for i in expression[1:]:
    for j in i.split("+"):
      sum -= int(j)
  return sum

print(solution())

File: 20200406/code.js
function solution(string) {
  sum = 0;
  arr = string.split("-");

  arr[0].split("+").forEach(v => sum += parseInt(v, 10));

  arr.slice(1).forEach(v => {
    v.split("+").forEach(k => sum -= parseInt(k, 10))
  });

  return sum;
}

console.log(solution("55-50+40"));
console.log(solution("55-50+40-10+20+20"));
